src/main/java/io/crate/jmx/recorder/RecorderRegistry.java

```diff
@@ -43,4 +44,8 @@ public static Recorder get(String name) {
 
     private RecorderRegistry() {
     }
+
+    public static void clear() {
```

Comment:
maybe rename to `clearRecorderInstances` or `resetRecorderInstance` as otherwise some could understand it as clear-the-registry